bin/phpstan

```diff
@@ -24,6 +25,13 @@ use Symfony\Component\Console\Helper\ProgressBar;
 
 	$devOrPharLoader = require_once __DIR__ . '/../vendor/autoload.php';
 	require_once __DIR__ . '/../preload.php';
+	$composer = ComposerHelper::getComposerConfig(getcwd());
+
+	if ($composer !== null) {
+		$vendorDirectory = ComposerHelper::getVendorDirFromComposerConfig(getcwd(), $composer);
+	} else {
+		$vendorDirectory = getcwd() . '/' . 'vendor';
```

Comment:
This still uses the hardcoded `vendor` But this was the only way I could think of.